config/main.py

```diff
@@ -454,7 +454,7 @@ def _change_hostname(hostname):
         run_command('sed -i "/\s{}$/d" /etc/hosts'.format(current_hostname), display_cmd=True)
         run_command('echo "127.0.0.1 {}" >> /etc/hosts'.format(hostname), display_cmd=True)
 
-def _clear_qos():
+def _clear_qos(namespace):
```

Comment:
why we need namespace here ? I don't think there is requirement of doing any qos operation for any paritcular namespace ?  If this not needed then we should hide namespace here Same comment on reload also,

Reply:
'namespace' is not required here. As long as we are consistent with other config commands, namespace option can be removed.